ibm/service/database/resource_ibm_database_postgresql_test.go

```diff
@@ -109,6 +109,22 @@ func TestAccIBMDatabaseInstancePostgresBasic(t *testing.T) {
 					resource.TestCheckResourceAttr(name, "tags.#", "1"),
 				),
 			},
+			{
+				Config: testAccCheckIBMDatabaseInstancePostgresGroupMigration(databaseResourceGroup, testName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckIBMDatabaseInstanceExists(name, &databaseInstanceOne),
+					resource.TestCheckResourceAttr(name, "name", testName),
+					resource.TestCheckResourceAttr(name, "service", "databases-for-postgresql"),
+					resource.TestCheckResourceAttr(name, "plan", "standard"),
+					resource.TestCheckResourceAttr(name, "location", acc.IcdDbRegion),
+					resource.TestCheckResourceAttr(name, "groups.0.memory.0.allocation_mb", "2048"),
```

Comment:
Will this value 2048 or 1024 since its for each member right?
Did the validation did not failed here

Reply:
`groups`  is a computed attribute that still populated by the V4 API / `bluemix-go` client library. It uses the total values used by the group, not per-member, so this check passes.